public/javascripts/discover.js

```diff
@@ -52,7 +53,8 @@ var discover = (function() {
                 langsExist: langsExist,
```

Comment:
what is langsExist? vs langShown or langFilter?
We could rename the variable to be readable.